modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -153,12 +155,14 @@ spec:
 <7> The interface name to synchronize from.
 <8> The interface to synchronize devices connected to the interface.
 <9> Specify system config options for the `phc2sys` service, for example `-a -r`. If this field is empty the PTP Operator does not start the `phc2sys` service.
-<10> Specify an array of one or more `recommend` objects which define rules on how the `profile` should be applied to nodes.
-<11> Specify the `profile` object name defined in the `profile` section.
-<12> Specify the `priority` with an integer value between `0` and `99`. A larger number gets lower priority, so a priority of `99` is lower than a priority of `10`. If a node can be matched with multiple profiles according to rules defined in the `match` field, the profile with the higher priority is applied to that node.
-<13> Specify `match` rules with `nodeLabel` or `nodeName`.
-<14> Specify `nodeLabel` with the `key` of `node.Labels` from the node object.
-<15> Specify `nodeName` with `node.Name` from the node object.
+<10> Scheduling policy for ptp4l and phc2sys processes. Default value is `SCHED_OTHER`. Use `SCHED_FIFO` on systems that support FIFO scheduling.
+<11> Integer value from 1-65 used to set FIFO priority for `ptp4l` and `phc2sys` processes. Required if `SCHED_FIFO` is set for `ptpSchedulingPolicy`.
+<12> Specify an array of one or more `recommend` objects which define rules on how the `profile` should be applied to nodes.
```

Comment:
ISG: Use "which", preceded by a comma, to introduce a nonrestrictive clause. Use "that", without a comma, to introduce a restrictive clause.
```suggestion
<12> Specify an array of one or more `recommend` objects that define rules on how the `profile` object should be applied to nodes.
```